src/i18n/locales/en/translation.json

```diff
@@ -334,6 +334,19 @@
           "connection_failed": "Connection failed. Are your credentials correct?"
         }
       },
+      "albyhub": {
+        "title": "Alby Hub",
+        "page": {
+          "instructions": "Paste a NWC connection string from Alby Hub",
+          "url": {
+            "label": "NWC connection string from Alby Hub",
+            "placeholder": "nostr+walletconnect://69effe..."
+          },
+          "errors": {
+            "connection_failed": "Connection failed. Is your Alby Hub online and app connection enabled?"
```

Comment:
What does "app connection enabled" mean in that context?

Reply:
mean that app connection still exists and usable